### PR TITLE
[test/plugin] immigrate canal scenario

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: test/plugin/scenarios/canal-scenario/bin/startup.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -Dcanal.host=${CANAL_HOST} -Dcanal.port=${CANAL_PORT} -jar ${agent_opts} "-Dskywalking.agent.service_name=canal-scenario" ${home}/../libs/canal-scenario.jar &
```
Comment (dmsolr) [inline test/plugin/scenarios/canal-scenario/bin/startup.sh]: We don't need to override configuration of agent  next time when the `server_name` is the same as `scenario_name`.

Comment (wu-sheng) [reply test/plugin/scenarios/canal-scenario/bin/startup.sh]: Let's keep this in the doc as recommendation.

Comment (arugal) [reply test/plugin/scenarios/canal-scenario/bin/startup.sh]: > We don't need to override configuration of agent next time when the `server_name` is the same as `scenario_name`.

deleted

**Comment by wu-sheng** [comment]
@kezhenxu94 strange coverage change. Just FYI.

**Pull request merged**